### ISSUE TITLE
New LuckPerms Webeditor Autofill

### ISSUE THREAD

**Issue by bartyrealms** [lucko/LuckPerms#1909]
Issue that I just experienced with the new LuckPerms webeditor (I have experienced this a bunch of times, I just usually catch it).

The old webeditor you had to press enter twice frequently when editing perms in the webeditor. With the new one, when adding a new permission to a group, you only have to press it once. This leads to that second enter press getting sent to whatever autofilled.

Well, if your mouse is in the top half of your screen, that autofilled permission is the * permission. This can be catastrophic to servers and is very dangerous. I recommend a rework of the autofill system so this does not occur again. The * permission being added is very discreet and hard to catch every time.

Example:
https://i.gyazo.com/4820781a0cc1a48cf3755651dead8289.mp4

In summary, pressing enter more than once when entering a permission can slip in a * permission, potentially causing a huge mess.

**Comment by Andre601** [comment]
https://github.com/lucko/LuckPermsWeb

**Comment by bartyrealms** [comment]
Whoops. Didn't realize they were on different places.
